mix.exs

```diff
@@ -41,6 +41,8 @@ defmodule DaisyUIComponents.MixProject do
       {:jason, "~> 1.4", only: [:dev, :test]},
       {:dialyxir, "~> 1.4", only: [:dev, :test], runtime: false},
       {:credo, "~> 1.7", only: [:dev, :test], runtime: false},
+      {:lazy_doc, "~> 0.5", only: :dev, runtime: false},
```

Comment:
for me would be better no dependencies since I want this library to contain the minimum required dependency amount. If lazy_doc could be run without config and through an escript, would be perfect for me. Similar to phx_new command, where you only need to install the script globally and you can use as a script to create a new phoenix application

Reply:
This dep is under `dev` scope so user is not paying the final cost of using `lazy_doc`